betterforms/multiform.py

```diff
@@ -17,7 +19,29 @@
 
 
 @python_2_unicode_compatible
-class MultiForm(object):
+class WhitelistedBaseForm(object):
```

Comment:
big docstring right here please. I like this idea though. It just needs heavy commenting.